.github/workflows/ci.yaml

```diff
@@ -372,7 +372,7 @@ jobs:
         id: cache-generated-source
         with:
           path: ${{ env.jit_generated_src_scheme }}
-          key: jit_generated_src_scheme-jit_${{env.jit_version}}-${{env.jit_causalhash}}
+          key: jit_generated_src_scheme-jit_${{env.jit_version}}-${{env.jit_causalhash}}-${{ hashFiles('**/ci.yaml') }}
```

Comment:
wondering if we should also hash `env.ucm` here given we rely on a transcript to generate the jit sources?